ui/src/app/machines/views/MachineList/MachineList.js

```diff
@@ -24,6 +24,30 @@ import { machine as machineSelectors } from "app/base/selectors";
 import { nodeStatus } from "app/base/enum";
 import { useWindowTitle } from "app/base/hooks";
 
+const formatStorageUnit = gb => {
```

Comment:
Could these live in a utility file somewhere? We use unit conversion functions in like a million places in the old angular code, each written slightly differently. Maybe we could start centralising some of these patterns.

Reply:
Yeah I think you're right. I pretty much just lifted these from the old codebase which was a bit lazy of me :roll_eyes: